tests/test_pytypes.cpp

```diff
@@ -70,6 +70,19 @@ TEST_SUBMODULE(pytypes, m) {
     m.def("dict_contains",
           [](const py::dict &dict, const char *val) { return dict.contains(val); });
 
+    // test_tuple
+    m.def("get_tuple", []() { return py::make_tuple(42, py::none(), "spam"); });
+
+#if PY_VERSION_HEX >= 0x03030000
+    // test_namespace
```

Comment:
Could you please insert the "simple" here as well (`test_simple_namespace`, `get_simple_namespace`)?

That will make it easier to pin-point everything to do with simple_namespace (useful for maintenance).

Reply:
Done.